lib/Doctrine/ORM/Tools/Pagination/Paginator.php

```diff
@@ -27,6 +27,7 @@
  * The paginator can handle various complex scenarios with DQL.
  *
  * @template-covariant T
+ * @implements IteratorAggregate<array-key, T>
```

Comment:
What does this fix?

Reply:
Ah I think I know: it fixes a warning about implementing IteratorAggregate w/o specifying the types. I have that same fix in a branch that fixes part of the issues detected by Psalm v5, I will share it soon.

Reply:
Yes. I found it as well while playing with Psalm 5.